web/src/user_groups.ts

```diff
@@ -278,6 +292,12 @@ export function get_realm_user_groups_for_dropdown_list_widget(
             if (!user_group) {
                 throw new Error(`Unknown group name: ${group.name}`);
             }
+
+            group.display_name = maybe_override_group_display_name_for_dropdown(
+                setting_name,
+                group,
+            );
+
```

Comment:
this section is used to replace `Nobody` with `Disable resolving topics` in dropdown. Is it fine, or you expect something better?